docs/getting_started.rst

```diff
@@ -134,6 +134,12 @@ Alternatively, to create just the output file or directory of one rule, run:
 
      snakemake -j<NUMBER_OF_CPU_CORES> <output file or folder>
 
+If needed, all calculated scenarios in the directory :file:`results` can be deleted with the following rule:
+
+::
+
+    snakemake -j1 clean
```

Comment:
Could you please give a distinction between Linux based systems (`rule clean` should apply to Linux but also to Darwin / macOS) and Windows system (`rule clean_on_win_sys`)?